### PR TITLE
 Customized volume panels[3/3]

### PR DESCRIPTION
**Pull request by Thatisjigen**

### CODE REVIEW

File: config/crdroid.mk [hunk 1]
```diff
@@ -177,3 +177,6 @@ PRODUCT_PACKAGES += \
     QStileTeardrop \
     QStileTriangle \
     QStileWavey
+
+# Customizable volume panels
+-include packages/apps/Plugins/plugins.mk
```
Comment (gwolf2u) [inline config/crdroid.mk:182]: where is this?

**Comment by gwolf2u** [comment]
also resolve conflicts

File: config/crdroid.mk (hunk 1 above)
Comment (Thatisjigen) [reply config/crdroid.mk:182]: https://github.com/PotatoProject/packages_apps_Plugins

**Comment by gwolf2u** [comment]
DNM for now

**Pull request closed without merging**